### PR TITLE
Consistently use bold in intro

### PR DESCRIPTION
**Pull request by YangFong**
### Description

Fix typos.

### CODE REVIEW

**Comment by Josh-Cena** [comment]
We apply bold to the description topic very inconsistently. I'm going to run a pass through the entire operators section and commit to your branch.

**Comment by Josh-Cena** [comment]
@mdn/yari-content-javascript I want to apply bold consistently to all opening paragraphs. Of the two styles, which one should we use?

```md
The **left shift assignment operator (`<<=`)** moves the specified amount of bits to the left and assigns the result to the variable.
```

```md
The **left shift assignment (`<<=`)** operator moves the specified amount of bits to the left and assigns the result to the variable.
```

I have strong arguments for using the second one.

1. For the global objects / methods section, we always only bold the code. For example, we say ``The **`push()`** method`` instead of ``The **`push()` method**``. This means "category labels" like "operator", "method", "keyword", "statement" should not be included in the bold range.
2. Most page titles do not include "operator" (although a few do), and it makes sense to make the bold consistent.
3. What's inside the brackets should be mutually substitutable with the text preceding it. For example, when I say "Open Web Docs (OWD) believes that...", it means both "Open Web Docs believes that..." and "OWD believes that...". However, if we put the ``(`<<=`)`` part after the "operator" word, it means the sentence would expand to "The `<<=` moves..." instead of "The `<<=` operator moves...", and the latter reads more naturally to me. ("`<<=` moves..." is natural as well, but the bold range implies that "the" is not part of the group.)

This is a really nitty point, but I don't want to settle on one side too easily without hearing what others think.

**Comment by YangFong** [comment]
In the operators section, most documents cover operator in bold, but as you said, the title does not include operator, I personally prefer to use the second style.

As with array methods, only the name of the method is bold; it is not necessary to include the method.

That said, bolding only the title content is a recommended way.

**Comment by Josh-Cena** [comment]
I'm also aware there are many pages that don't apply bold at all (like the one you touched), so it's a good chance to make everything consistent as well.

**Comment by wbamberg** [comment]
> I have strong arguments for using the second one.
> ...

For me those are pretty convincing :).

File: files/en-us/web/javascript/reference/operators/class/index.md [hunk 1]
```diff
@@ -14,9 +14,7 @@ browser-compat: javascript.operators.class
 
 {{jsSidebar("Operators")}}
 
-The **class expression** is one way to define a class. Similar to [function expressions](/en-US/docs/Web/JavaScript/Reference/Operators/function), class expressions can be named or unnamed. If named, the name of the class is local to the class body only.
-
-JavaScript classes use prototype-based inheritance.
```
Comment (Rumyra) [inline files/en-us/web/javascript/reference/operators/class/index.md:19]: Was there a reason to remove this line?

**Comment by Rumyra** [comment]
This looks like a great improvement - thanks @YangFong @Josh-Cena 

I only have one query - otherwise I'm happy to merge this 👍

File: files/en-us/web/javascript/reference/operators/class/index.md (hunk 1 above)
Comment (Josh-Cena) [reply files/en-us/web/javascript/reference/operators/class/index.md:19]: This page mainly talks about the syntax, so semantics information looks out of context on this page. There are so many things interesting to talk about with classes—"prototypical inheritance" is by far not worthy to be put in the intro, not to mention that private properties actually do not participate in prototypical inheritance. It's better for readers to read everything about classes in the class reference.

Comment (Rumyra) [reply files/en-us/web/javascript/reference/operators/class/index.md:19]: Thanks for the explanation @Josh-Cena :)

**Review by Rumyra — APPROVED** [review]

**Pull request merged**